10/umbraco-deploy/deployment-workflow/README.md

```diff
@@ -11,7 +11,7 @@ Umbraco Deploy uses a deployment model that relies on Git and Umbraco Deploy cor
 If your project contains a Staging environment, deployments will be made from Development to Staging and then from Staging to Live.
 {% endhint %}
 
-![Left to right model](<../../../umbraco-cloud/deployment/images/left-to-right (1) (1) (1) (1) (1) (1) (1) (1) (1).png>)
+![Left to right model](images/left-to-right.png)
```

Comment:
I've cleaned up some of the duplicated images (that contains the repeated `(1)` in the filenames), but it looks like there are a lot more that can be cleaned up: https://github.com/search?q=repo%3Aumbraco%2FUmbracoDocs%20%22(1)%22&type=code.

Also: some images are placed in the `.gitbook/assets/images/` directory, instead of the `images/` directory relative to the documentation files...

Reply:
Thanks for the cleanup, @ronaldbarendse 💪We are aware of the "(1)" in the filenames; it's automatically added by GitBook when we upload images through Gitbook. Also, GitBook stores images in the `.gitbook/assets/images/` folder, which is why you see both that directory and the `images/` directory.